local-volume/README.md

```diff
@@ -148,7 +148,9 @@ spec:
   local:
     path: /mnt/disks/ssd1
 ```
-
+Please replace "my-node" with the name of kubernetes node which is hosting this
```

Comment:
You also should replace `name`, `capacity.storage`, `storageClassName`, and `local.path` fields too.